pkg/controller/kfservice/resources/knative/service.go

```diff
@@ -87,15 +98,22 @@ func (c *ServiceBuilder) CreateEndpointService(kfsvc *v1alpha2.KFService, endpoi
 		//TODO create transformer
 		return nil, nil
 	case constants.Explainer:
-		explainerSpec := &kfsvc.Spec.Default.Explainer
+		explainerSpec := kfsvc.Spec.Default.Explainer
+		predictorHost := constants.DefaultPredictorServiceName(kfsvc.Name) + "." + kfsvc.ObjectMeta.Namespace
```

Comment:
if transformer is on spec, the call needs to direct there?

Reply:
If we are calling "explainerServiceName" name later, would make sense to call "predictorServiceName" - to be consistent?

Reply:
Yes, will add transformer is it exists